packages/subgraph/scripts/generate-introspection-json.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+if [ -f .env ]
+then
+  export $(cat .env | sed 's/#.*//g' | xargs)
+fi
+
+if [ -z "$NETWORK_NAME" ] || [ -z "$SUBGRAPH_NAME" ] || [ -z "$GRAPH_KEY" ] || [ -z "$SUBGRAPH_VERSION" ]
+then
+    echo "env variables are not set properly, exiting..."
+    exit -1
+fi
+
+# Exit script as soon as a command fails.
+set -o errexit
+# Up docker
+docker-compose -f docker/docker-compose.yml up -d
+# Wait for containers to start
+sleep 10
```

Comment:
may be this delay is not stable and not enough,
what we could do is to loop with delay 2 seconds, and once the docker is up then continue with the rest of the code